src/detectors/DRICH/DRICH.cc

```diff
@@ -34,6 +34,9 @@ extern "C" {
     digi_cfg.enablePixelGaps = true;
     digi_cfg.pixelSize       = 3.0; // [mm]
     digi_cfg.safetyFactor    = 0.7;
+    digi_cfg.enableNoise     = true;
```

Comment:
Let's keep the noise injection off for now in this PR, so the IRT PID algorithm continues to have the same performance. Checking how well IRT does with noise is the next immediate step, however.
```suggestion
    digi_cfg.enableNoise     = false;
```